tasks/connectors/ms_defender_tvm/ms_defender_tvm.rb

```diff
@@ -170,6 +130,26 @@ def run(opts)
             end
 
             machine_id = vuln.fetch("deviceId")
+            fqdn = vuln.fetch("deviceName")
+
+            # Get the asset details & craft them into a hash
+            asset = {
+              "external_id" => machine_id,
+              "fqdn" => fqdn,
+              "hostname" => fqdn.split(".")[0],
+              "os" => vuln.fetch("osPlatform"),
+              "os_version" => vuln.fetch("osVersion"),
+              "first_seen" => vuln.fetch("firstSeenTimestamp"),
+              "last_seen" => vuln.fetch("lastSeenTimestamp")
+            }
+
+            # Construct tags
+            tags = []
+            tags << "MSDefenderTvm"
+            tags << "rbacGroup: #{vuln.fetch('rbacGroupName')}" unless vuln.fetch("rbacGroupName").nil?
```

Comment:
Did you intentionally leave out:
`        tags << "riskScore: #{machine.fetch('riskScore')}" unless machine.fetch("riskScore").nil?
          tags << "exposureLevel: #{machine.fetch('exposureLevel')}" unless machine.fetch("exposureLevel").nil?
          tags << "TVM Agent Version: #{machine.fetch('agentVersion')}" unless machine.fetch("agentVersion").nil?
          tags.concat(machine.fetch("machineTags")) unless machine.fetch("machineTags").nil?`
that was in set_clien_data()?

Reply:
yes the data doesn't exist in the new api